### PR TITLE
Allow FactoryFinder to try to load class with jakarta api class's cla…

### PR DESCRIPTION
**Pull request by jimma**
…ssloader

### CODE REVIEW

**Comment by jimma** [comment]
@lukasj  Can you please review ?

**Comment by jamezp** [comment]
@jimma It looks like the JavaDoc year needs to be updated as there is a validation failure for that.

That said this fix looks okay to me.

**Comment by jimma** [comment]
@jamezp  Thanks for review, James !  Already fixed this copyright year issue.

**Comment by jimma** [comment]
@lukasj  Can you please review this PR ?

File: api/pom.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 1997, 2021 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 1997, 2022 Oracle and/or its affiliates. All rights reserved.
```
Comment (lukasj) [inline api/pom.xml:4]: these changes were not needed as the gh build action failure is caused by the way the PR build works

**Review by lukasj — COMMENTED** [review]
@jimma I have no idea what this PR is trying to fix. Do you expect me to figure it out myself from the code change?

**Comment by jmehrens** [comment]
Looks like this PR is not needed as this was fixed in:

https://github.com/jakartaee/jaf-api/commit/64b41f6c31c997de3d8ec6cced9b3b6c51394ec0

@lukasj Any chance I can get added to the activation team since I'm already on the mail team?  I'm unable to close this issue.

**Comment by lukasj** [comment]
closing per last comment

> @lukasj Any chance I can get added to the activation team since I'm already on the mail team? I'm unable to close this issue.

@jmehrens  it's not that simple here but let me see what can be done..

**Pull request closed without merging**